v22.2/take-and-restore-locality-aware-backups.md

```diff
@@ -117,29 +117,43 @@ To restore from a specific backup, use [`RESTORE FROM {subdirectory} IN ...`](re
 
-{% include_cached copy-clipboard.html %}
-~~~ sql
-> BACKUP INTO LATEST IN
-	  ('s3://us-east-bucket?COCKROACH_LOCALITY=default', 's3://us-west-bucket?COCKROACH_LOCALITY=region%3Dus-west');
-~~~
+There is different syntax for taking an incremental backup depending on where you need to store the backups:
 
-{{site.data.alerts.callout_info}}
-When [restoring from an incremental locality-aware backup](#restore-from-an-incremental-locality-aware-backup), you need to include **every** locality ever used, even if it was only used once.
+- To append your incremental backup to the full backup in the [`incrementals` directory](take-full-and-incremental-backups.html#backup-collections):
 
-It is recommended that the same localities be included for every incremental backup in the series of backups; however, only the `default` locality is required.
-{{site.data.alerts.end}}
+	{% include_cached copy-clipboard.html %}
+	~~~ sql
+	BACKUP INTO LATEST IN
+		('s3://us-east-bucket?COCKROACH_LOCALITY=default', 's3://us-west-bucket?COCKROACH_LOCALITY=region%3Dus-west');
+	~~~
 
-And if you want to explicitly control where your incremental backups go, specify the subdirectory in the `BACKUP` statement:
+	{{site.data.alerts.callout_info}}
+	When [restoring from an incremental locality-aware backup](#restore-from-an-incremental-locality-aware-backup), you need to include **every** locality ever used, even if it was only used once.
 
-{% include_cached copy-clipboard.html %}
-~~~ sql
-> BACKUP INTO {subdirectory} IN
-		('s3://us-east-bucket?COCKROACH_LOCALITY=default', 's3://us-west-bucket?COCKROACH_LOCALITY=region%3Dus-west');
-~~~
+	It is necessary that the same localities be included for every incremental backup in the series of backups; however, only the `default` locality is required.
+	{{site.data.alerts.end}}
+
+- If you want to explicitly control the subdirectory for your incremental backups:
+
+	{% include_cached copy-clipboard.html %}
+	~~~ sql
+	BACKUP INTO {subdirectory} IN
+			('s3://us-east-bucket?COCKROACH_LOCALITY=default', 's3://us-west-bucket?COCKROACH_LOCALITY=region%3Dus-west');
+	~~~
+
+	To view the available subdirectories, use [`SHOW BACKUPS`](restore.html#view-the-backup-subdirectories).
+
+- If you are creating an incremental backup using the [`incremental_location`](backup.html#options) option to send your incremental backups to a different location, you must include the same number of locality-aware URIs for the full backup destination and the `incremental_location` option:
+
+	{% include_cached copy-clipboard.html %}
+	~~~ sql
+	BACKUP INTO LATEST IN
+		('s3://us-east-bucket?COCKROACH_LOCALITY=default', 's3://us-west-bucket?COCKROACH_LOCALITY=region%3Dus-west') WITH incremental_location = ('s3://us-east-bucket-2?COCKROACH_LOCALITY=default', 's3://us-west-bucket-2?COCKROACH_LOCALITY=region%3Dus-west');
```

Comment:
it might be worth mentioning this current limitation: https://github.com/cockroachdb/cockroach/issues/82912

Reply:
Ah, yes, that's a good idea. I will add that in. Thank you!